composer.json

```diff
@@ -49,5 +50,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "php-http/discovery": false
```

Comment:
Disable all plugins with  `"*": false` rather than specifying this one specifically.

Reply:
@emodric This has been fixed after rebasing the branch to master.